server/routes/users/index.js

```diff
@@ -7,25 +8,45 @@
 module.exports = async function (fastify, options) {
   const User = require('../../models/User');
 
+  /**
+   * Get all users
+   * Should be Admin only
+   */
```

Comment:
Not necessarily important, but more of a question, should we set a standard for documentation throughout the codebase? Such as consistently writing doxygen above function definitions?

Reply:
Yes but using JSDOC, It's already added as part of the eslint rules